webapi/src/main/scala/org/knora/webapi/routing/v1/ResourcesRouteV1.scala

```diff
@@ -544,6 +590,68 @@ object ResourcesRouteV1 extends Authenticator {
                         loggingAdapter
                     )
             }
+
+        } ~  path("v1" / "resources" / "xml" ) {
+            post {
+                entity(as[NodeSeq]) {
+                    xml => requestContext =>
+                    val userProfile = getUserProfileV1(requestContext)
+
+                    val projectId = "http://data.knora.org/projects/DczxPs-sR6aZN91qV92ZmQ"
+                    val root = xml.head
+                    val apiRequestID = UUID.fromString("26106dcd-865a-4c81-b0e8-914e46939e70")
+                    val resourcesToCreate = root.child
```

Comment:
Also, please make an XSL Schema for this XML, and use it to validate the XML before processing it.